### ISSUE TITLE
Cannot create a tensor proto whose content is larger than 2GB - Different RGB 128x128 Dataset

### ISSUE THREAD

**Issue by Ahmedest61** [naturomics/CapsLayer#17]
On a different RGB dataset with 128x128 size, after extracting features and labels, in **get_batch_data()** when **tf.train.slice_input_producer([trX, trY])()** is called, this mentioned error comes! Any one found the solution of it? :(
